samples/play-kml-tour/src/main/java/com/esri/arcgismaps/sample/playkmltour/components/PlayKmlTourViewModel.kt

```diff
@@ -50,11 +49,11 @@ class PlayKmlTourViewModel(application: Application) : AndroidViewModel(applicat
 
     // add elevation data
     private val surface = Surface().apply {
-        elevationSources.add(ArcGISTiledElevationSource("https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"))
+        elevationSources.add(ArcGISTiledElevationSource(application.getString(R.string.world_terrain_service)))
```

Comment:
I think it'll be more readable on the web or code view if URLs are written out in code, rather than hidden in strings file